src/ontology/mondo-edit.obo

```diff
@@ -308158,7 +308160,7 @@ is_a: MONDO:0003778 {source="PMID:11941303", source="https://orcid.org/0000-0001
 is_a: MONDO:0003847 {source="https://orcid.org/0000-0002-5002-8648"} ! hereditary disease
 is_a: MONDO:0005046 {source="https://orcid.org/0000-0001-8216-5084"} ! immune system disorder
 relationship: curated_content_resource https://search.clinicalgenome.org/kb/conditions/MONDO:0013576 {source="MONDO:CLINGEN"}
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/5716 {source="MONDO:mim2gene_medgen", source="OMIM:614102"} ! IGKC
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/5716 {source="MONDO:mim2gene_medgen"} ! IGKC
```

Comment:
There is a HGNC identifier so while the OMIM data does not have the HGNC information I have added this into the "protected" file so it will not be removed in the future.